rwgame/RWGame.cpp

```diff
@@ -457,7 +457,7 @@ void RWGame::tick(float dt)
 		}
 		
 		// Clean up old VisualFX
-		for( int i = 0; i < world->effects.size(); ++i )
+		for( ssize_t i = 0; i < static_cast<ssize_t>(world->effects.size()); ++i )
```

Comment:
@JayFoxRox Yes, or destroyEffect should return an iterator (a la `erase()`).

But I'm far too lazy for that, please ignore it for now. ;)

@paulsapps Because the `--i` on `i=0` will overflow.